### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
PiperOrigin-RevId: 463690521

### CODE REVIEW

**Comment by trav3711** [comment]
Converted all of the scoring tool testing samples from the old building config format to the new building config format.

File: tools/validators/ontology_validator/setup.py [hunk 1]
```diff
@@ -29,5 +29,5 @@
     description='',
     packages=find_packages(),
     install_requires=['absl-py', 'pyglib', 'pyyaml>=5.3', 'argparse'],
-    python_requires='>=3.9',
```
Comment (charbull) [inline tools/validators/ontology_validator/setup.py:32]: this must stay

File: tools/validators/ontology_validator/yamlformat/validator.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 from os import path
 import sys
 
-from arg_parser import CreateParser
+from google3.third_party.digitalbuildings.tools.validators.ontology_validator.yamlformat.arg_parser import CreateParser
```
Comment (charbull) [inline tools/validators/ontology_validator/yamlformat/validator.py:25]: this is not correct, update the copybara mapping rules?

File: tools/validators/instance_validator/README.md [hunk 3]
```diff
@@ -36,9 +36,10 @@ Note: as of the current development stage, you must clone the entire repository
 
 The validator supports a telemetry validation mode. When this mode is enabled, the validator will listen on a provided pubsub subscription for telemetry messages, and validate the message contents against the instance configuration. It is recommended that you first use the instance validator with telemetry validation mode disabled, and then enable it after that passes.
 
-If you would like to use the telemetry validation mode, you must provide the `--subscription` parameter, and you must either:
```
Comment (charbull) [inline tools/validators/instance_validator/README.md:39]: this is Nour's pull request, you are overriting it.

**Review by charbull — CHANGES_REQUESTED** [review]
The scoring on github is the source of truth as Adam worked direclty on github.
The instance validator (Nour added a pull request) we need to make sure we are not deleting things

File: tools/validators/ontology_validator/yamlformat/validator.py (hunk 2 above)
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/validator.py:25]: Will update copy.bara.sky to fix this and send a CL.

File: tools/validators/ontology_validator/setup.py (hunk 1 above)
Comment (trav3711) [reply tools/validators/ontology_validator/setup.py:32]: Done.

File: tools/validators/instance_validator/README.md (hunk 3 above)
Comment (trav3711) [reply tools/validators/instance_validator/README.md:39]: Reverted README to Nours PR.

**Comment by trav3711** [comment]
@charbull I reverted scoring/ to commit: https://github.com/google/digitalbuildings/commit/5a1cc22553a141f6023466066806485bef376e61#diff-bb8f5823415f4636cce78ac39b3ee1594754f2c63f51a7df372ce27fd10c4c18

I then had to update the test cases to match our current instance validator requirements for the new building config format.

@ahemphill does the Scoring tool code look good to you?

**Review by charbull — CHANGES_REQUESTED** [review]
Commented

**Review by charbull — CHANGES_REQUESTED** [review]
for the scoring/ folder, the github is the source of truth so all these changes from g3 to github regarding the scoring folder must be reverted and then synched back to g3.

**Review by sypks — APPROVED** [review]
lgtm

**Review by sypks — APPROVED** [review]

**Review by charbull — APPROVED** [review]

**Pull request merged**